high_card.rb

```diff
@@ -1,5 +1,62 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+cards = []
+
+suits.each do |suit|
+  ranks.each do |rank|
+    cards.push([rank, suit])
+  end
+end
+
+deck = cards.shuffle
+num_of_players = 0
+while (num_of_players < 22)
+  puts 'Enter a player name. If there are not more players, enter "done".'
+  input = gets.chomp
+  if input == "done"
+    break
+  end
+  players.push(input)
+num_of_players += 1
+end
+
+puts "OK! Here's who's in the game: #{players}"
```

Comment:
When printing out an array, I'd use `.inspect`. Otherwise they're `puts`ed out onto separate lines, and I like 1 line per `puts`. For example: `#{players.inspect}`